packages/frontapp/src/app/components/UserSearch/index.tsx

```diff
@@ -38,22 +38,16 @@ const UserSearch: FC<IProps> = ({ setActive }) => {
     navigate(RoutesTypes.READERS);
   };
 
-  const radioBtnHandler = (
-    attributes: Attribute[],
-    type: string,
-    value: string
-  ) => {
-    if (type === 'radio') {
-      if (attributes[0].title === value) {
-        attributes[1].checked = false;
-        attributes[2].checked = false;
-      } else if (attributes[1].title === value) {
-        attributes[0].checked = false;
-        attributes[2].checked = false;
-      } else {
-        attributes[0].checked = false;
-        attributes[1].checked = false;
-      }
+  const radioBtnHandler = (attributes: Attribute[], value: string) => {
+    if (attributes[0].title === value) {
+      attributes[1].checked = false;
+      attributes[2].checked = false;
+    } else if (attributes[1].title === value) {
+      attributes[0].checked = false;
+      attributes[2].checked = false;
+    } else {
+      attributes[0].checked = false;
+      attributes[1].checked = false;
```

Comment:
I don't like how this handler looks like. What if we want to add more params to the filter? Check how the radiogroup handler look like on the `Settings` page and rewrite it. Radio button uncheckes other values by default when we are selecting a new one